engine/ui/IgeUiLabel.js

```diff
@@ -145,7 +146,7 @@ var IgeUiLabel = IgeUiElement.extend({
 	
 	nativeFont: function (val) {
 		if (val !== undefined) {
-			this._fontEntity.nativeFont(val);
+			// this._fontEntity.nativeFont(val);
```

Comment:
Are you sure this is correct? The nativeFont method will no longer work as a setter.